courses/serializers/v2/departments.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import Prefetch, Q, Count
+from mitol.common.utils import now_in_utc
+from rest_framework import serializers
+
+from courses.models import CourseRun, Department
+
+
+class DepartmentSerializer(serializers.ModelSerializer):
+    """Department model serializer"""
+
+    class Meta:
+        model = Department
+        fields = ["id", "name"]
+
+
+class DepartmentWithCountSerializer(DepartmentSerializer):
+    """CourseRun model serializer that includes the number of courses and programs associated with each departments"""
```

Comment:
nit: this isn't a CourseRun serializer

Reply:
copy pasta. oops. I'll fix before I merge.